packages/app/src/logger.ts

```diff
@@ -0,0 +1,12 @@
+import { Console } from 'console';
+import { app } from 'electron';
+import { createWriteStream } from 'fs-extra';
+
```

Comment:
alternatively https://www.npmjs.com/package/winston is a good option as well - very flexible and powerful